mysite/myapp2/templates/myapp2/client_detail.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}

```

Comment:
А еще ни один шаблон не рендерится, так как не верно указан путь для расширения.
```suggestion
{% extends 'myapp2/base.html' %}

```